v3/transports/webrtc.md

```diff
@@ -9,4 +9,13 @@ When an endpoint supports this it includes in it's `paths` a `{"type":"webrtc"}`
 
 All packets must be [cloaked](../e3x/cloaking.md) and are sent as base64 encoded message events over a data channel.
 
+## JSON
 
+Example JSON:
+
+```json
+{
+    "signal": "TODO:What goes here",
```

Comment:
Due to the nature of WebRTC and signals being dynamic, the path json is nothing other than a `{"type":"webrtc"}` to signal support of the transport.  There may be some helpful static hints that could be included but I haven't encountered them yet, as everything is negotiated via the signaling during the actual peer connection setup :)